### DISCUSSION TITLE
How to combine exceptions from different countries

### DISCUSSION THREAD

**Discussion by stevenbrix** [getsentry/sentry-dotnet#3305] [Q&A]
We have a global set of users, the same set of exceptions is logged for them, however since the messages are in different languages, the sentry website doesn't group them. Are there any tricks/tips on how to make sure that the same exception (based off HRESULT) are all grouped?

**Comment by jamescrosswell** [comment]
Hm, tricky. You can [control of the fingerprinting](https://docs.sentry.io/platforms/dotnet/usage/sdk-fingerprinting/) to have it group things however you want... I can't see an easy solution to this particular problem though. Possibly checking if the Exception type is the same and then checking the line number from the top frame in the stack trace... however I think that's simplifying somewhat and won't always work.
